### PR TITLE
Made a change.

### PR DESCRIPTION
**Pull request by millebesso**
Yada yada

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # methodology
 This repo contains information about the working method.
+
+More descriptions here.
```
Comment (olabackstrom) [inline README.md]: Somewhat fluffy, please improve

File: Method.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+There's even more.
```
Comment (olabackstrom) [inline Method.txt]: Test

File: README.md [hunk 3]
```diff
@@ -1,2 +1,4 @@
 # methodology
 This repo contains information about the working method.
```
Comment (millebesso) [inline README.md]: This should have been removed

File: README.md (hunk 1 above)
Comment (olabackstrom) [inline README.md]: I think #1  should be adressed

**Comment by millebesso** [comment]
I'm closing this as it works now.

**Pull request merged**